herald/src/main/java/io/heraldprox/herald/sensor/analysis/aggregates/Mean.java

```diff
@@ -32,6 +32,9 @@ public void map(Sample<T> value) {
 
     @Override
     public Double reduce() {
+        if (count == 0) {
```

Comment:
0 == count see style guide https://heraldprox.io/contribute/style

Reply:
- Updated all instances of "== CONSTANT" to "CONSTANT ==" across all classes
- Passes unit tests